docs/modules/kubernetes/pages/helm-upgrade-guide.adoc

```diff
@@ -0,0 +1,165 @@
+= Helm Upgrade Guide
+
+Before upgrading, review the Hazelcast's xref:hazelcast:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[Compatibility Guarantees].
+
+Due to a link:https://github.com/helm/charts/issues/7803[bug in the Helm Project], the helm upgrade command does not allow modifications to the spec.volumeClaimTemplates property in stateful sets. This results in some limitations for persistence in the Management Center and any hotRestart/persistence using `vilumeClaimTemplates` internally.
+The following sections provide some methods of addressing these limitations.
+
+NOTE: Although we have tested common scenarios without hotRestart (persistence) and the Management Center persistence feature enabled, we cannot test every upgrade scenario. If you encounter issues, please contact xref:operator:ROOT:index.adoc#support[Support].
+
```

Comment:
```suggestion
== Upgrade Management Center
```

Reply:
Following steps does not upgrade management center. To upgrade Hazelcast properly, we first disable management center and then upgrade Hazelcast. At last we re-enable management center. So this title is wrong.

Reply:
OK. I felt the structure was unclear; but I think I see now what was bothering me. 

If we remove the command from step 4, and expand the list item text to "Upgrade your installation, and re-enable Management Center, as described in the Upgrade Hazelcast section below" I think that sorts it.

What do you think?

Reply:
Yes, that make sense to me, thank you.